packages/dev/docs/src/docs.css

```diff
@@ -447,14 +447,25 @@ h2.sectionHeader {
 .pageHeader {
   position: fixed;
```

Comment:
do we want to change this to not be `position: fixed` for mobile views? Right now the top of fullscreen/fullscreen takeover modals is still getting cut off in smaller viewports:
![image](https://user-images.githubusercontent.com/9661127/84703302-e2eef180-af0c-11ea-83d8-7df5b1e942bf.png)

Reply:
ah, thanks, didn't know about this issues